src/Sonarr.Api.V3/Logs/LogController.cs

```diff
@@ -3,10 +3,11 @@
 using NzbDrone.Core.Instrumentation;
 using Sonarr.Http;
 using Sonarr.Http.Extensions;
-
+using Sonarr.Http.REST.Filters;
 namespace Sonarr.Api.V3.Logs
 {
     [V3ApiController]
+    [TypeFilter(typeof(LogDatabaseDisabledActionFilterAttribute<PagingResource<LogResource>>))]
```

Comment:
Since this is the only place we want to short circuit it completely, replace the attribute with a check in `GetLogs`